src/test/java/jp/ikedam/jenkins/plugins/updatesitesmanager/ManagedUpdateSiteJenkinsTest.java

```diff
@@ -99,78 +94,6 @@ private File getResource(String name) throws URISyntaxException, FileNotFoundExc
         return new File(url.toURI());
     }
     
-    public void testDoPostBack() throws URISyntaxException, IOException, GeneralSecurityException

```

Comment:
Though we no longer override `doPostBack`, this test is still valid as an integration.
Is it that this test does't work with this change?